samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -9,11 +9,12 @@
 #include <drivers/gpio.h>
 #include <stdio.h>
 #include <net/lwm2m.h>
-#include <modem/nrf_modem_lib.h>
-#include <settings/settings.h>
-
 #include <net/lwm2m_client_utils.h>
 #include <net/lwm2m_client_utils_fota.h>
+#include <modem/nrf_modem_lib.h>
```

Comment:
I think most changes like this were introduced by both of us adding the same things in slightly different places. I will try to avoid creating diffs like this in the future though